isis/src/control/objs/BundleUtilities/BundleObservationSolveSettings.cpp

```diff
@@ -47,38 +47,189 @@ namespace Isis {
   }
 
 
-#if 0
-  /**
-   * Construct this BundleSettings object from XML.
-   *
-   * @param bundleSettingsFolder Where this settings XML resides - /work/.../projectRoot/images/import1
-   * @param xmlReader An XML reader that's up to an <bundleSettings/> tag.
-   */
-  BundleObservationSolveSettings::BundleObservationSolveSettings(
-                                                             FileName xmlFile,
-                                                             Project *project,
-                                                             XmlStackedHandlerReader *xmlReader) {
-
+  BundleObservationSolveSettings::BundleObservationSolveSettings(const PvlGroup &scParameterGroup) {
     initialize();
-    QString xmlPath = xmlFile.expanded();
-    QFile qXmlFile(xmlPath);
-    if (!qXmlFile.open(QFile::ReadOnly) ) {
-      throw IException(IException::Io,
-                       QString("Unable to open xml file, [%1],  with read access").arg(xmlPath),
-                       _FILEINFO_);
+
+      // group name must be instrument id
+    m_instrumentId = (QString)scParameterGroup.nameKeyword();
+
+    // If CKDEGREE is not specified, then a default of 2 is used
+    if (scParameterGroup.hasKeyword("CKDEGREE")) {
+      m_ckDegree = (int)(scParameterGroup.findKeyword("CKDEGREE"));
     }
 
-    QXmlInputSource xmlInputSource(&qXmlFile);
-    xmlReader->pushContentHandler(new XmlHandler(this, project));
-    xmlReader->setErrorHandler(new XmlHandler(this, project));
-    bool success = xmlReader->parse(xmlInputSource);
-    if (!success) {
-      throw IException(IException::Unknown,
-                       QString("Failed to parse xml file, [%1]").arg(xmlPath),
-                        _FILEINFO_);
+    // If CKSOLVEDEGREE is not specified, then a default of 2 is used -------jwb----- why ??? why not match camsolve option ???
```

Comment:
Old comment?

Reply:
This is still true as it's set in the initialize() call above. Could remove Jeannie's comment/make the change.